### PR TITLE
Release-cut 20.2 Revised

### PR DESCRIPTION
**Pull request by hemantt**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/projects/RDM/versions/36004

### Change description ###
**Revised list of Version 20.2** 

Bug

    [RDM-8698] - Elasticsearch exact match queries on phone numbers with spaces never return results
    [RDM-9824] - ChallengeQuestion tab issues

Feature

    [RDM-7484] - Add Support for Concurrent Execution of Test Suites

Task

    [RDM-9200] - FTA for Swagger Documentation and API Specs - Data Store API
    [RDM-9533] - Improve performance of LS Sweeper
    [RDM-9694] - CCD Logstash to index cases in state DRAFT in Elasticsearch
    [RDM-9696] - Create Liquibase Changeset For Additional Indexes
    [RDM-9741] - Revert Changes of RDM-9338
    [RDM-9910] - Fix Logstash "ghost cases" indexing issue
    [RDM-10321] - Enable Persistent Queue for Logstash
    [RDM-10389] - Release 20.2 Perftest/Demo deployment and Testing

Improvement

    [RDM-9343] - Elasticsearch sorts are case-sensitive

TechDebt

    [RDM-9939] - Move Organisation Policy validation from DefaultValidateCaseFieldsOperation to a dedicated validator
    [RDM-9954] - Remove no longer needed "defaultValue" field from API response

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: charts/ccd-data-store-api/values.preview.template.yaml [hunk 1]
```diff
@@ -16,8 +16,13 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
+<<<<<<< HEAD
     DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-811.service.core-compute-preview.internal/
     USER_PROFILE_HOST: http://ccd-user-profile-api-pr-339.service.core-compute-preview.internal/
+=======
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
+    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-335.service.core-compute-preview.internal/
+>>>>>>> f5846dfe7... Update def store wiring
```
Comment (danlysiak) [inline charts/ccd-data-store-api/values.preview.template.yaml:25]: Needs resolving 😄

File: src/main/resources/db/changelog/db.changelog-master.xml [hunk 2]
```diff
@@ -39,4 +39,5 @@
     <include file="db/changelog/db.changelog-RDM-8463.xml"/>
     <include file="db/changelog/db.changelog-RDM-9287.xml"/>
     <include file="db/changelog/db.changelog-RDM-9505.xml"/>
```
Comment (danlysiak) [inline src/main/resources/db/changelog/db.changelog-master.xml:41]: Think we're missing a line in here for RDM-9696 (two files up from this one)?

**Review by danlysiak — CHANGES_REQUESTED** [review]
Couple of minor things

**Review by danlysiak — APPROVED** [review]

**Review by mario-paniccia — APPROVED** [review]
👍

**Pull request merged**